### DISCUSSION TITLE
Build for Aarch64 platform

### DISCUSSION THREAD

**Discussion by pa-gaurav** [intel/llvm#13038] [Q&A]
How can I build it for Aarch64 Linux platform. is it possible? if it is can I have the steps to follow the build.

**Comment by KornevNikita** [comment]
Sorry, I don't have an answer for you, but have to move it to the discussions as that's not a bug report.

**Comment by rodburns** [comment]
See the answer here, the instructions will be the same for any Aarch64 platform https://github.com/intel/llvm/discussions/14586#discussioncomment-10117407
